### ISSUE TITLE
New Use Case: Olivocerebellum Scaffold

### ISSUE THREAD

**Issue by AliceGem** [cnr-ibf-pa/hbp-bsp-issues#529]
## New Use Case
  
Aspect | Detail
------------ | -------------
Summary | Simulations of an olivocerebellar scaffold network with E-GLIF point neurons (second release of cerebellar scaffold microcircuit in [Casali et al., 2019]).
Usecase Group | Brain Area Circuit In Silico Experiments
Expert | Alice Geminiani
Scientific User |
Deadline | March 2020
Target audience | Developers, Computational Neuroscientists
Target interface | Notebook
HPC Requirements |
Dependencies |
Nominal runtime |
  

## List of features
*What does your use case do? What activities can users perform while using it? What makes it different from similar use cases? List the main functions that you will build into your product here. Also specify the priority ‘mandatory’, ‘important, ‘nice to have’.*

The current use case runs simulations of an olivocerebellar Spiking Neural Network, using the NEST simulator (version 2.18.0). The scaffold is imported from an HDF5 in the storage and used to create nodes and connections in NEST. Then, three input stimuli are provided: a constant background at 4 Hz and a 250-ms Poisson generator at 40 Hz to the glomeruli, a 10-ms burst at 500 Hz to the Inferior Olive neurons. Then, a 1760-ms simulation is run. Simulation results (raster plots and PSTH of different neural populations) can be visualized.

    
## Acceptance Criteria
*Define here the acceptance tests to evaluate the use case’s compliance with the requirements as defined above. Also possible end users for testing can be included here.*

Runnable in Notebook.
Output frequency of simulated neural populations in physiological ranges.
Documented.

  
### System
*Does this use case require any special system requirements (any special software package)? Does this use case require HPC access? If so, which ones? List detailed system requirements here.*

Longer simulation times would require using MPI or HPC access.
  
### Performance
*What are additional performance requirements for this use case? What time should it take to run from start to end?*

Full notebook run (including NEST installation and network simulation) - not in parallel - requires about 1 hour.

**Comment by alex4200** [comment]
@AliceGem Do you have any updates on this task?

**Comment by AliceGem** [comment]
@alex4200 I have opened the pull request now, thanks!

**Comment by alex4200** [comment]
@AliceGem Any news on this task?

**Comment by antonelepfl** [comment]
If IIRC this UC is online on Production https://collab.humanbrainproject.eu/#/collab/1655/nav/66856 Under the Cerebellum Category

**Comment by alex4200** [comment]
@AliceGem  The last missing piece is a short documentation for the guidebook: 

https://collab.humanbrainproject.eu/#/collab/1655/nav/18580

**Comment by AliceGem** [comment]
Ok @alex4200! In which branch should I add it?

**Comment by alex4200** [comment]
@AliceGem I suggest you create a pull request to the repo: https://github.com/lbologna/hbp-sp6-guidebook or contact @lbologna directly.

**Comment by alex4200** [comment]
@AliceGem Do you have any updates on this issue?

**Comment by AliceGem** [comment]
Hi @alex4200, I opened the pull request yesterday.

**Comment by lbologna** [comment]
Hello @alex4200, @AliceGem,

I inserted the new page with some minor changes to text and links.

Here's the link
https://lbologna.github.io/hbp-sp6-guidebook/online_usecases/brain_area_circuit_in_silico_experiments/cerebellum/functional_simulation_with_eglif/functional_simulation_with_eglif.html

Could  you please have a look before moving to prod?
Thank you.

**Comment by AliceGem** [comment]
@lbologna thanks! I checked the links and they are ok, the rest is fine!

**Comment by lbologna** [comment]
@alex4200 @AliceGem, I just did the minor changes as discussed during the call.

**Comment by alex4200** [comment]
Then ok from my side to push to prod

**Comment by lbologna** [comment]
> Then ok from my side to push to prod

Ok, thanks.
Pushed to prod. Closing the ticket.
